pytest.ini

```diff
@@ -6,3 +6,4 @@ python_paths = lib
 markers =
   data_manager: marks test as a data_manager test
   tool: marks test as a tool test
+  local_screenshotting: marks test as a screenshot producer
```

Comment:
How do you feel about `screenshot` ?
If you want to discrimate local and remote you should add a second `local` and/or `remote` marker.
These markers can be combined arbitrarily.

Reply:
Ok, will do